.github/workflows/backend-lint.yml

```diff
@@ -88,14 +96,12 @@ jobs:
           #      `getconf ARG_MAX` - max # bytes of args + environ for exec()
           pre-commit run --files ${{ steps.files.outputs.all_files }}
 
-      # Apply fixed files via pull request review comments
-      - name: Apply any pre-commit fixed files as Pull Request review comments
-        if: github.ref != 'refs/heads/master' && steps.changes.outputs.backend == 'true' && always()
-        continue-on-error: true
-        uses: getsentry/action-git-diff-suggestions@main
+      # If working tree is dirty, commit and update if we have a token
+      - name: Commit any eslint fixed files
+        if: steps.token.outcome == 'success' && github.ref != 'refs/heads/master' && steps.changes.outputs.backend == 'true' && always()
```

Comment:
What is `always()`? I read the docs and I still not grasp it.

> Always returns true, even when canceled. A job or step will not run when a critical failure prevents the task from running. For example, if getting sources failed.

Reply:
The default behavior for GHA is for a failed step, every subsequent step will be cancelled. Using `always()` will ensure that the step runs even if the previous step fails. The word is a bit confusing, because you would think that regardless of the other boolean comparisons, that `always()` means it will *ALWAYS* run (but that's not the case).